tests/unit/plugins/test_go.py

```diff
@@ -110,6 +66,30 @@ def test_schema(self):
                 "Expected {!r} to be included in properties".format(expected),
             )
 
+        # Check go-channel
+        go_channel = properties["go-channel"]
+        for expected in ["type", "default"]:
+            self.assertTrue(
+                expected in go_channel,
+                "Expected {!r} to be included in 'go-channel'".format(expected),
+            )
+
+        go_channel_type = go_channel["type"]
+        self.assertThat(
+            go_channel_type,
+            Equals("string"),
+            'Expected "go-channel" "type" to be "string", but '
```

Comment:
... but here we're checking for double quotes ... (to be continued)

Reply:
This is help text for when an assertion fails, black, the autoformatter we use would have changed the outer quotes to be double if there hadn't been any inner single quotes.

This is all rather legacy.